### ISSUE TITLE
TRAINING YOLOV8 IN GPU

### ISSUE THREAD

**Issue by Liam7754** [ultralytics/ultralytics#8585]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Question

I tried to run yolo8 in GPU but it's not working. I have run into this ERROR

![Screenshot (401)](https://github.com/ultralytics/ultralytics/assets/125567675/5dfe501c-be42-4e0d-bf1d-b913646c48b9)

I also tried to check whether my torch, torchaudio, and torchvision support CUDA or not but the result returned positive

![Screenshot (403)](https://github.com/ultralytics/ultralytics/assets/125567675/2f96cb02-425d-460a-a845-7bf495d54101)

Can someone pls tell me what to do?

### Additional

no response

**Comment by Burhan-Q** [comment]
@Liam7754 

1. You need to verify you have the correct version of `torchvision` installed. There are specifically compatible versions and it _must_ also include the same CUDA version you have installed for `torch`. See this [compatibility table](https://github.com/pytorch/vision#installation)
1. The best way to get a report of your current setup is to run `yolo checks` from the command line with your environment enabled
1. Also I recommend using python 3.11 as 3.12 is still new and may not be compatible with all libraries

We also have a Discord server where you can get support from other community members as well <a href="https://ultralytics.com/discord"><img alt="Discord" src="https://img.shields.io/discord/1089800235347353640?logo=discord&logoColor=white&label=Discord&color=blue"></a>
